app/views/shared/org_selectors/_text_only.html.erb

```diff
@@ -0,0 +1,38 @@
+<%# ISSUE93: in case the change of plan-text field for funder affecting other pages,
+this view is specially created to force saving a new funder without warning message pop-up %>
+
+<%
+# Whether or not the org selection is required
+required = required || false
+# The label to use
+label = label || _("Organisation")
+# Allows the hidden id field to be renamed for instances where there are
+# multiple org selectors on the same form
+id_field = id_field || :org_id
+presenter = OrgSelectionPresenter.new(orgs: orgs, selection: default_org)
+%>
+
+<%= form.label :org_name, label %>
+<%= form.text_field :org_name, class: "form-control autocomplete",
+                            value: presenter.name,
+                            spellcheck: true,
+                            authenticity_token: true,
+                            data: { source: "" } %>
+<span class="autocomplete-help sr-only" role="status"></span>
+<div id="<%= SecureRandom.hex(6) %>_ui-front" class="ui-front"></div>
+
+<%# sources contains an array of Org names %>
+<%= form.hidden_field :org_sources, value: presenter.select_list %>
+<%# crosswalk contains an array of hashes that contain the Org name, id,
+    identifiers like ROR and other info used by the OrgSelectionService %>
+<%= form.hidden_field :org_crosswalk, value: presenter.crosswalk %>
+<%# gets updated with the matching record from crosswalk when the user
+    selects or enters something %>
+<% if form.object[id_field]&.to_s =~ /[0-9]+/ || form.object[id_field].nil? %>
+  <% val = presenter.crosswalk_entry_from_org_id(value: form.object[id_field]) %>
+  <%= form.hidden_field id_field, value: val, class: "autocomplete-result",
+                                              autocomplete: "off" %>
+<% else %>
+  <%= form.hidden_field :org_id, class: "autocomplete-result",
+                                 autocomplete: "off" %>
+<% end %>
```

Comment:
Does the controller expect these values to be present like this on the view? If not we could probably do without them since we will not be providing autocomplete suggestions to the user.

Reply:
@lagoan I personally prefer to keep these code, because removing them also requires removing/large modification of the AJAX call for the org-selection JS method, which will trigger the org_selectable and create_org ruby methods. Since the AJAX call is shared among all org selection actions (not just the plan part), I hope to avoid touching them because we still need org auto-completion for other pages. Thus I feel leaving these code here is ok to achieve our goal (temporary remove the auto-complete just for funder) while not creating too much impact to other part. 

How do you think? I could also write a distinct JS just for this change (something like text_only.js), modifying ruby code and removing this part, but the functionality is already implemented by leaving these pieces here.

Reply:
You are right, we should avoid unnecessary changes to the base code.